ParkingLotTest/UnitTest1.cs

```diff
@@ -6,10 +12,303 @@ namespace ParkingLotTest
     public class UnitTest1
     {
         [Fact]
-        public void Test1()
+        public void Parking_Boy_Can_Park_A_Car_Into_Parking_Lot_Test()
+        {
```

Comment:
seems that "expectedReducedNUmberOfPosition, expectedIncreasedNumberOfCar" is not need to fullfill AC1